src/Microsoft.Performance.SDK.Runtime/CustomDataSourceReference.cs

```diff
@@ -13,60 +13,55 @@ namespace Microsoft.Performance.SDK.Runtime
     /// <summary>
     ///     Creates a <see cref="AssemblyTypeReferenceWithInstance{T, Derived}"/> where T is <see cref="ICustomDataSource"/> and Derived is <see cref="CustomDataSourceReference"/>.
     /// </summary>
-    public sealed class CustomDataSourceReference
+    public abstract class CustomDataSourceReference
```

Comment:
>CustomDataSourceReference [](start = 26, length = 25)

I know it would be breaking, but do want to rename this CustomDataSourceReferenceBase, to be consistent with our other class names?